backend/app/models/account.rb

```diff
@@ -29,4 +29,8 @@ def slug_preview
     set_slug unless slug.present?
     slug
   end
+
+  def approved?
+    investor&.approved? || project_developer&.approved?
```

Comment:
I'm not sure why we have approved on investor/project_developer and not directly on account, maybe that is just a mistake? @agnessa

Reply:
Yeah, hmm, I agree with you, if `Account` model is used to save data which are same for Investor and Project Developer, then `approved` attributes should be moved to it. If you all of us agree, I can refactor code this way :)

Reply:
agree

Reply:
Yes, better to move it. Seems more logical to have account.approved directly there.